tests/ui/lifetimes/explicit-lifetime-arguments-warning-72278.rs

```diff
@@ -1,3 +1,4 @@
+// https://github.com/rust-lang/rust/issues/72278
```

Comment:
Suggestion: please add a backlink to the tracking issue for this FCW: https://github.com/rust-lang/rust/issues/42868 and mention the lint name `late_bound_lifetime_arguments`